app/views/spree/checkout/_state_change.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if target && target.state_changes.any? %>
```

Comment:
@rishabhguptarishi you can remove this commit from this PR, do it in a separate branch and merge.